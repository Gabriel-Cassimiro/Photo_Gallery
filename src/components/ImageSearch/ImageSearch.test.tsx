/**
 * @jest-environment jsdom
 */

import { render } from "@testing-library/react"
import { ImageSearch } from "./index"

describe("Test Search Box component", () => {
	it("Should render correctly", () => {
		const { container } = render(<ImageSearch />)
		expect(container).toMatchSnapshot()
	})
})
