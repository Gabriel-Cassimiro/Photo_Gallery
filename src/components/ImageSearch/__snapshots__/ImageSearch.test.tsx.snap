// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Search Box component Should render correctly 1`] = `
<div>
  <div
    class="max-w-xs rounded overflow-hidden "
  >
    <form
      class="w-full max-w-sm"
    >
      <div
        class="flex items-center border-b-2 border-teal-500 py-1"
      >
        <input
          class="appearance-none bg-transparent border-none w-full text-gray-700 mr-3 py-1 px-2 leading-tight focus:outline-none"
          placeholder="Search Photo Term..."
          type="text"
          value=""
        />
        <button
          class="shrink-0 bg-teal-500 hover:bg-teal-700 border-teal-500 hover:border-teal-700 text-sm border-4 text-white py-1 px-2 rounded"
          type="submit"
        >
          Search
        </button>
      </div>
    </form>
  </div>
</div>
`;
